docs/metrics/accelerate.md

```diff
@@ -64,9 +64,11 @@ average(incident resolved timestamp - incident created timestamp)
 Pulse uses the maximum value over a period of time to display aggregate times to recover.
 
 !!! note
-    **Pulse first calculates the performance metrics per [system](../cli/cli.md#before-you-begin)**, and later aggregates the metrics by time interval when displaying them.
+    If you are reporting incidents to Pulse using the [PagerDuty integration](../one-click-integrations/pagerduty-integration.md), the [Pulse CLI](../cli/cli.md), or the [Pulse Ingestion API](https://ingestion.pulse.codacy.com/v1/api-docs):
```

Comment:
@mrfyda I believe the original note doesn't apply when Incidents data is detected from Bitbucket or GitHub, so I added this condition. Can you please validate if this is correct? 🙏

Reply:
as far as I know, the source of the events doesn't impact the way the metrics are calculated. the GitHub and Bitbucket integrations will generate Incident events that will be indistinguishable from those coming from PagerDuty or the API/CLI

with that in mind I'd revert this change

pinging @afsmeira to be sure

Reply:
For some context, I considered that this message is targeting PagerDuty or the API/CLI because it's highlighting the need to report both deployments and incidents associated with the correct systems. And I assumed that when Incidents data comes from GitHub or Bitbucket this highlight wouldn't be needed.

But you are probably right! Let's wait for @afsmeira input. 👍

Reply:
Thanks, @afsmeira, that's a great suggestion. 👍 
Updated!